cloud/aws/common/config.go

```diff
@@ -23,19 +23,28 @@ import (
 )
 
 type AwsApiConfig struct {
-	Domains []string
+	Description string
+	Domains     []string
 }
 
 type AwsImports struct {
 	// A map of nitric names to ARNs
 	Secrets map[string]string
+	Buckets map[string]string
+}
+
+type BatchComputeEnvConfig struct {
+	MinCpus       int      `mapstructure:"min-cpus"`
+	MaxCpus       int      `mapstructure:"max-cpus"`
+	InstanceTypes []string `mapstructure:"instance-types"`
 }
 
 type AwsConfig struct {
 	ScheduleTimezone                      string `mapstructure:"schedule-timezone,omitempty"`
 	Import                                AwsImports
 	Refresh                               bool
 	Apis                                  map[string]*AwsApiConfig
+	BatchComputeEnvConfig                 *BatchComputeEnvConfig `mapstructure:"batch-compute-env,omitempty"`
```

Comment:
The GCP config uses the key:`batch-compute`. Should we change this for consistency?

```suggestion
	BatchComputeEnvConfig                 *BatchComputeEnvConfig `mapstructure:"batch-compute,omitempty"`
```

Reply:
for GCP its a bit different, the `ComputeEnvironment` is a proper resource in AWS. Happy to look at updating the name in GCP instead. But wanted the key to be a reflection of the resource that was being created in the users AWS account.